### ISSUE TITLE
The output from the command allocate error

### ISSUE THREAD

**Issue by ikasoumen** [kactus-io/kactus#66]
### Description
Hi, 
I add *60.2MB* Sketch file and try to initial commit, but an error was happening.  
Here is the detail: 
```
The output from the command could not fit into the allocated stdout buffer.
Set options.maxBuffer to a larger value than 10485760 bytes
```

Someone knows how to fix it?

### Version

**Kactus version:** [0.2.11]

**OS version:** [macOS 10.12.6]

**Comment by mathieudutour** [comment]
hum I'll try to find a fix. In the meantime, you can run `git add . && git commit -m "first commit"` in your terminal (that you can open from Kactus). Then it should work, it's only for the first commit because there are tons of files.

**Comment by mathieudutour** [comment]
this will be fixed in 0.3.1 :tada:
